ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
NB: any change here should also be made in Roxie's implementation of checkPersistUptoDate() in ccdcontext.cpp